src/controllers/flowsheet.controller.ts

```diff
@@ -282,3 +282,20 @@ export const getOnAir: RequestHandler = async (req, res, next) => {
     next(e);
   }
 };
+
+export const changeOrder: RequestHandler<object, unknown, { entry_id: number; new_order: number }> = async (req, res, next) => {
+  const { entry_id, new_order } = req.body;
+
+  if (entry_id === undefined || new_order === undefined) {
+    res.status(400).json({ message: 'Bad Request: entry_id and new_order are required' });
+  } else {
+    try {
+      const updatedEntry = await flowsheet_service.changeOrder(entry_id, new_order);
+      res.status(200).json(updatedEntry);
+    } catch (e) {
+      console.error('Error: Failed to change order');
+      console.error(e);
```

Comment:
Do we get some sort of performance and error monitoring system with the runtime that we're using? We should think about that if we don't.

Reply:
We do - the AWS/Docker backend reports things to a log file, I believe, but we'll want to confirm with Adrian on whether it is saved and for how long.

Reply:
I haven't implemented anything for managing persisted logs/ clearing logs, but logs should be stored in a file /var/lib/docker/containers/<container_id>. I'd like to set something more sophisticated up, but haven't had a chance.